src/job_handler_plugins/local_container/__init__.py

```diff
@@ -48,7 +48,7 @@ def start(self) -> str:
         self.client.containers.run(
             image=full_image_name,
             command=custom_command,
-            name=self.job.entity["name"],
+            name=self.job.entity.get("name", "local-job"),
```

Comment:
So as we discussed, should be possible to run multiple jobs simultaneously.
So adding a short 5-8 char random suffix would help.